components/sync/service/brave_sync_service_impl.h

```diff
@@ -116,6 +117,15 @@ class BraveSyncServiceImpl : public SyncServiceImpl {
   void UpdateP3AObjectsNumber();
   void OnGetTypeEntitiesCount(const TypeEntitiesCount& count);
 
+  // IdentityManager::Observer implementation.
+  // Override with an empty implementation.
```

Comment:
please provide an explanation in the comments

Reply:
Updated https://github.com/brave/brave-core/pull/25971/commits/83f3830438d211a41729eba320afb8617de0a6e6